test/internet/test-http-dns-fail.js

```diff
@@ -4,7 +4,6 @@
  * should trigger the error event after each attempt.
  */
 
-var common = require('../common');
```

Comment:
Just saw that the rule allows exceptions per `varsIgnorePattern`, so that could be anoter option.